packages/server/src/mail/mail-services.controller.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  Controller,
+  Get,
+  Param,
+  Patch,
+  UseGuards,
+  HttpException,
+  HttpStatus,
+  Body,
+} from '@nestjs/common';
+import { MailService } from './mail.service';
+import { JwtAuthGuard } from 'guards/jwt-auth.guard';
+import { MailServiceModel } from './mail-services.entity';
+import { MailServiceWithSubscription } from '@koh/common';
+import { User } from 'decorators/user.decorator';
+import { UserModel } from 'profile/user.entity';
+import { UserSubscriptionModel } from './user-subscriptions.entity';
+@UseGuards(JwtAuthGuard)
```

Comment:
```suggestion
@UseGuards(JwtAuthGuard, EmailVerifiedGuard)
```

Probably a good idea to make sure their email is verified before they can subscribe to email service ;P